lib/myapp_web/live/product_live/index.ex

```diff
@@ -40,19 +41,19 @@ defmodule MyappWeb.ProductLive.Index do
     {:noreply, assign(socket, :products, fetch_products())}
   end
 
-  def handle_info({_, result}, socket) do
+  def handle_info({_, {:result, component_id, _http_result}}, socket) do
     require Logger
     Logger.error("Received")
 
-    send_update(FormComponent, id: "product-form", refreshing: false)
+    send_update(FormComponent, id: component_id, refreshing: false)
```

Comment:
This is the secret sauce here. The parent live view gets the component id here so that it can route to it. Notably, this is not the same thing as the DOM id of the form.